build.gradle

```diff
@@ -149,6 +149,9 @@ dependencies {
         exclude group: 'org.ow2.asm', module: 'asm-tree'
     }
 
+    // used for output encoding of config descriptions
+    implementation group: 'org.owasp.encoder' , name: 'encoder', version: '1.2.3'
```

Comment:
when is this used? I might have missed it but doesn't come up in a quick search

Reply:
it will be used in later PRs.